scrapy/commands/__init__.py

```diff
@@ -14,15 +15,15 @@
 class ScrapyCommand:
 
     requires_project = False
-    crawler_process = None
+    crawler_process: Any = None
```

Comment:
Could it be `Optional[CrawlerProcess]` instead of Any?

Reply:
(here and in the other commands as well)

Reply:
I originally tried it when mypy activated for this file and decided that `crawler_process` was of type `None`, but it exposes potential type errors in other parts of the code:
```
scrapy\commands\genspider.py:137: error: Item "None" of "Optional[CrawlerProcess]" has no attribute "spider_loader"  [union-attr]
```